docs/samples/bookinfo.md

```diff
@@ -28,136 +28,76 @@ There are 3 versions of the reviews microservice:
 
 The end-to-end architecture of the application is shown below.
 
-![Bookinfo app_noistio](./img/bookinfo/noistio.svg)
+![Bookinfo app_noistio]({{site.bareurl}}/docs/samples/img/bookinfo/noistio.svg)
 
 This application is polyglot, i.e., the microservices are written in different languages.
 
 {% endcapture %}
```

Comment:
Make sure to remove al the {%capture%} stuff, and update the document front-matter, as per my post in the docs slack channel this weekend.